pypeit/reduce.py

```diff
@@ -1203,37 +1216,58 @@ def local_skysub_extract(self, global_sky, sobjs, spat_pix=None, model_noise=Tru
         # overkill since nothing is extracted
         self.sobjs = sobjs.copy()  # WHY DO WE CREATE A COPY HERE?
 
+        base_gpm = self.sciImg.select_flag(invert=True)
+
+        if not self.sciImg.shot_noise:
```

Comment:
I'm not sure I'm following why this was added?

Reply:
No longer needed.  Removed.